lib/elixir/src/elixir_erl.erl

```diff
@@ -21,11 +21,13 @@ debug_info(elixir_v1, _Module, none, _Opts) ->
   {error, missing};
 debug_info(elixir_v1, _Module, {elixir_v1, Map, _Specs}, _Opts) ->
   {ok, Map};
-debug_info(erlang_v1, _Module, {elixir_v1, Map, Specs}, _Opts) ->
-  {Prefix, Forms, _, _} = dynamic_form(Map),
+debug_info(erlang_v1, Module, {elixir_v1, Map, Specs}, _Opts) ->
+  Data = elixir_module:data_table(Module),
+  {Prefix, Forms, _, _} = dynamic_form(Map, Data),
   {ok, Prefix ++ Specs ++ Forms};
-debug_info(core_v1, _Module, {elixir_v1, Map, Specs}, Opts) ->
-  {Prefix, Forms, _, _} = dynamic_form(Map),
+debug_info(core_v1, Module, {elixir_v1, Map, Specs}, Opts) ->
+  Data = elixir_module:data_table(Module),
```

Comment:
I'm having some problems with the chunks in protocols, I believe is related to the way that the `debug_info` is saved. I'm checking this